README.md

```diff
@@ -168,40 +171,80 @@ python3 trainer.py --config configs/<training-config>.yaml
-::
-    pip install nox
+The development mode for all model projects is based on a tool, called *[Nox](https://nox.thea.codes/en/stable/)*.
 
-### Experiment a use case
+*Nox* is a python build tool, that allows to define targets (in a similar way that Make does), to simplify command execution in development and CI/CD pipelines. By default, each *Nox* session (equivalent to a make target) is executed in a specific virtualenv that ensure code partitioning and experiments reproducibility.
 
-Several Nox targets allow to handle easily an experimentation of any use case on a demo dataset and configuration.
+#### Installation
 
-Choose the _Model Collection_ use case you want to experiment and go in.
-::
-    cd weather_forcast/gwd
-There you can display the list of the available targets with
-::
-    nox --list
+*Nox* can be installed using *pip*.
 
-Please note, some of them are experimentation oriented, while other ones are CI/CD oriented.
+```
+pip install nox
+```
+
+To install the version of *Nox* used in a existing model project please refer to the ``requirements.txt`` file of the project.
+
+```
+pip install $(grep nox== requirements.txt)
+```
 
-*Coming soon ...*
+#### Usage
 
-You can launch a demo training on the model use case with ``nox -s train``
+In each model project directory the ``noxfile.py`` file implements all the *Nox* sessions available (note most of them are actually imported from a global file in ``tools/nox/nox_ref_file.py``).
 
-### Development mode
+In the model project you can display the list of the available targets with
+```
+nox --list
+```
+
+The main sessions you will use for development will be :
+- ``dev_dependencies``
+- ``tests``
+- ``lint``
+
+Note these seesions are used to run the CI/CD and build the docker image of a model project. So using them and making sure they succeed are major steps on the road of proposing your contribution.
+
+#### Create the proper python development environment
 
 The nox target are also very useful to launch generic command during development phase.
 
-#### Run unit tests
+##### Run unit tests
 
 You can run the whole unit test suite of a use case, using ``pytest``, with ``nox -s tests``.
 This target also prints out the coverage report and save a xml version in ``.ci-reports/``.
 
-#### Run linting
+##### Run linting
+
+You can run the python linting of the code use case, using ``flake8``, with ``nox -s lint``.
+
+
+### Existing model project
+
+Please note, the maintenance policy of the repository is to consider a model project freezed from the moment when it is intergrated. It means you can open an issue, but the only major ones (bugs breaking the functionality or leading the model to be scientifically unrelevant) will be treated, but no upgrade nor refactoring of the existing model code will be adressed.
```

Comment:
but neither the upgrade nor the refactoring of the existing model code will be adressed.